src/libsyncengine/syncpal/syncpal.cpp

```diff
@@ -549,6 +549,14 @@ bool SyncPal::vfsForceStatus(const SyncPath &path, bool isSyncing, int progress,
     return _vfsForceStatus(_syncDbId, path, isSyncing, progress, isHydrated);
 }
 
+bool SyncPal::vfsCleanUpStatuses() {
+    if (!_vfsCleanUpStatuses) {
+        return false;
```

Comment:
Logging?

Reply:
I kept the same logic as the others callback (see all method starting with `vfs` in `syncpal.cpp`).